### ISSUE TITLE
PHP Error: Invalid plugin name: in "About"

### ISSUE THREAD

**Issue by texxasrulez** [roundcube/roundcubemail#7364]
After updating RC to 1.4.4, and I click on "About" in Roundcube GUI upper left hand corner, I get the following error in RC logs for a couple plugins, all error same except for **various_plugins** which is the plugin name of course ... 

PHP Error: Invalid plugin name: **various_plugins**/**various_plugins** in /var/www/htdocs/mail/program/lib/Roundcube/rcube_plugin_api.php on line 297 (GET /mail/?_task=settings&_action=about)

Let me know if I need to provide anything else. 
First time issue submitter, long time lover of your work.

**Comment by alecpl** [comment]
What do you have in `$config['plugins']`?

**Comment by texxasrulez** [comment]
Here is my plugin list:

`$config['plugins']` = array('elastic4mobile',
							'account_details',
							'additional_imap',
							'additional_smtp',
							'acl',
							'additional_message_headers',
							'authres_status',
							'calendar',
							'carddav',
							'contextmenu',
							'contextmenu_folder',
							'easy_unsubscribe',
							'emoticons',
							'fail2ban',
							'folder_info',
							'html5_notifier',
							'markasjunk',
							'new_user_dialog',
							'newmail_notifier',
							'nextcloud',
							'odfviewer',
							'password',
//							'pwtools',
							'pdfviewer',
							'quota',
							'sauserprefs',
							'savepassword',
//							'scheduled_sending',
							'tls_icon',
							'vcard_attach',
							'vcard_attachments'					
							`);
`

I have a few popular plugins and a few I maintain and the ones commented out are a WIP .... 
Let me know if you would like me to disable some plugins and help nail this down and I will be happy to do so ...

**Comment by alecpl** [comment]
Ok, I was able to reproduce the error " PHP Error: Invalid plugin name: pear/net_sieve...". 

Looks like we call rcube_plugin_api::get_info() on all plugin dependencies, which does not make sense. Problem is that now such a call logs a warning. We'll have to remove such calls.

**Comment by texxasrulez** [comment]
Awesome. I submitted a valid bug. WHOO HOO. I am excited to help out. 
Thank you for your work. Like I said, I have been a long time user, pretty much since Roundcube launched and love what you all do. 
Roundcube is by far my most favorite OSS project out there. It made me teach myself a little programming so I could wirte my own plugins for it and resurrect outdated abandoned plugins. Your efforts has made me a better "Nerd", as I prefer to be called, and I appreciate it.

On a personal note, I see your personal site ends with .pl for Poland. My Grandmother is Polish. DeMetrick (sp) was her maiden name. It was changed to Metrack when her parents immigrated to the US. Just a personal note to ya, who knows, we could be kinfolk.

I do thank you for your hard work. I develope OSS projects myself, just small ones, so I know all about the hours and dedication it takes to do what you all do and I want to thank you all for your efforts and devotion. I just hope one day to write something this awesome.

I hope you and yours are staying safe out there during these crazy times.
